super-cool-game/Assets/Scripts/Player/PlayerController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+
+// This is gonna be used to send multiplayer commands too.
+public class PlayerController : MonoBehaviour {
+    /* Translates inputs from PlayerInput into move commands
+     * to be sento to PlayerMovement 
+     */
+
+    private PlayerMovement pm;
+    private float speed = 3.0f;
```

Comment:
```speed = 4.0f``` looks better to me than 3. GabeN approves, too.

Consider settings the variables public instead of private to have direct access to them in the editor for testing changes run-time.

Reply:
Edited speed to 4.0f